### DISCUSSION TITLE
Allow links on the notifications page to open new tabs/windows

### DISCUSSION THREAD

**Discussion by Raivshard** [civitai/civitai#243] [Ideas]
Allow links on the notifications page to open new tabs/windows

Yeah, that's pretty much it. As they are, they only open up in the current view, and do not act as normal links, so you can't even mouse down and drag.

**Comment by JustMaier** [comment]
Hey, this has been added and is live now 👍

**Reply by Raivshard** [reply]
Cool! Now, can you please add CTRL-click functionality to that process? Right-clicking and selecting from a list is a pain.
